tools/secure_notebook.py

```diff
@@ -19,13 +27,14 @@
 import json
 
 
-def create_self_signed_cert(cert_dir, keyfile, certfiile):
+def create_self_signed_cert(cert_dir, keyfile, certfile):
     """
-    If datacard.crt and datacard.key don't exist in cert_dir, create a new
-    self-signed cert and keypair and write them into that directory.
+    Create a self-signed `keyfile` and `certfile` in `cert_dir`
+
+    Abort if one of the keyfile of certfile exist.
     """
 
-    if not exists(join(cert_dir, certfiile)) \
```

Comment:
I would write the logic in the opposite order: `if exists: raise, else: write()`. Less composite negatives make the code much easier to follow, I think, even if the exception goes first, which is a bit unusual.

But just my take on it, OK if you disagree.